### DISCUSSION TITLE
Geometria Engine:

### DISCUSSION THREAD

**Discussion by TheNachoBIT** [Geometria-Engine/Geometria#6] [Memes & Off Topic]
![geo-meme](https://user-images.githubusercontent.com/43283743/191998145-0038b3f3-1523-47c4-8584-b04da3acd35b.png)

**Comment by HastagGuigui** [comment]
Jokes aside, I wonder how would a geometria game perform on console 🤔

**Reply by TheNachoBIT** [reply]
I mean, you can *try* atleast to port the engine to a console as long as its old enough lmao

The most recent consoles you can actually port games to without paying are the PS3 and the Xbox 360, both PS4/PS5 devkits are still worth +USD$2000

**Reply by HastagGuigui** [reply]
Good thing I have an Xbox 360
I would have also thought about the 3Ds but last time I checked the devkit is straight up no longer available (tho I haven't created an actual Nintendo developer account so I don't actually know)
